README.md

```diff
@@ -1,6 +1,19 @@
-# worldcubeassociation.org [![Build Status](https://github.com/thewca/worldcubeassociation.org/actions/workflows/ruby.yml/badge.svg?event=push)](https://github.com/thewca/worldcubeassociation.org/actions/workflows/ruby.yml) [![Coverage Status](https://coveralls.io/repos/github/thewca/worldcubeassociation.org/badge.svg?branch=master)](https://coveralls.io/github/thewca/worldcubeassociation.org?branch=master)
+<h1 align="center">WorldCubeAssociation Website</h1>
+<div align="center">
+  <img height="120" src="client/public/favicon.png" />
```

Comment:
This file doesn't exist in our repo

Reply:
Downloaded the png from WCA Live and added it to our repo - looked at the existing files we have, but all the logos have "World Cube Association" at the bottom, and I don't believe .md files can render .ico, which is why I didn't use our favicon.